wagtail/wagtailadmin/templates/wagtailadmin/pages/listing/_page_title_choose.html

```diff
@@ -5,7 +5,7 @@
 {% endcomment %}
 
 <h2>
-    {% if page.can_choose %}
+    {% if page.can_choose and not page.is_root %}
```

Comment:
In that case, am I correct in thinking the problem is https://github.com/torchbox/wagtail/blob/master/wagtail/wagtailadmin/views/chooser.py#L48 ?

Checking the page is of a specific class isn't enough in this case.